Lowercase_To_Uppercase_C/main.c

```diff
@@ -0,0 +1,58 @@
+#include <stdio.h>
+
+char FromLowerToUppercase(char letter)
+{
+    char uppercaseLetter = letter - 32;
+    return uppercaseLetter;
+}
+
+void PrintAnswer(int a, char character)
+{
+    char uppercaseLetter;
+
+    switch (a)
+    {
+        case 1:
+            uppercaseLetter = FromLowerToUppercase(character);
+            printf("Uppercase of %c", character);
+            printf(" is %c\n", uppercaseLetter);
+            break;
+        case 2:
+            printf("You already entered uppercase letter: %c\n", character);
+            break;
+        case 3:
+            printf("Your entered character is not a letter: %c\n", character);
+            break;
+    }
+}
+
+int IsLetterOrUppercaseLetter(char letter)
```

Comment:
Bad naming. I don't understand what it does. The method name should describe the action.